Framework/Backend/services/notification.js

```diff
@@ -69,81 +69,25 @@ class KafkaConnector {
    * @param {string} message message to be sent
    * @returns {Promise}
    */
-  async _send(topic, message) {
+  async send(tag, title, url, extra) {
```

Comment:
If the method is to be used within this class only, please keep the naming convention to use `_`

Reply:
Nope, this is to be used from outside, eg. by Bookkeeping